### ISSUE TITLE
Image being created on final file but not displayed on the docx

### ISSUE THREAD

**Issue by jmdiego** [open-xml-templating/docxtemplater-image-module#66]
Hi,

My code for nodejs:

```javascript
var opts = {};
opts.centered = false;
opts.getImage = function(tagValue, tagName) {
    return new Buffer(tagValue.replace(/^data:image\/\w+;base64,/, ""), 'base64').toString('binary');
};
opts.getSize = function(img,tagValue, tagName) {
    return [400, 200];
};
var imageModule = new ImageModule(opts);
var reportContent = fs.readFileSync("./document.docx","binary");
var reportDoc = new Docxtemplater().attachModule(imageModule).load(reportContent).setOptions({delimiters:{start:'{{',end:'}}'}});
reportDoc.setData({...});
reportDoc.render();
var reportFinalBuffer = reportDoc.getZip().generate({type : "nodebuffer"});
```

The issue, as described in the title, is that if I convert the final .docx to zip, I can find the image, but when I open the final .docx it won't show the image, it's also removing all contents after the image var {{%reportImage}}.

What am I doing wrong?

**Comment by edi9999** [comment]
Can you do this: 

```
buffer = new Buffer(tagValue.replace(/^data:image\/\w+;base64,/, ""), 'base64')

fs.writeFileSync("image", buffer);
```

And see if you can open the image in an image viewer. Also, what is the type of the file ? You can use the UNIX file to get that information. 

I don't see a good reason why it doesn't show up, maybe it has to do that it deletes text after.

The content that is deleted is only in that paragraph, or is it in the whole document ? If it is in the paragraph, it is expected because the module replaces the tag with a block image. 

If it replaces the whole text even after the current paragraph, my guess is that the xml is somehow incorrect, can you post the contents of word/document.xml of the generated document ?

**Comment by jmdiego** [comment]
The file created with fileSync is working, I can even open it from paint.  The content that gets removed is everything after the image tag, I'm posting a resulting document.  Thanks for the assistance!!!
 
[generatedDoc.zip](https://github.com/open-xml-templating/docxtemplater-image-module/files/422817/generatedDoc.zip)

**Comment by edi9999** [comment]
Hi,

I dont see an image in the docx that you sent after unzipping. Can you also send your template and your json data?

**Comment by jmdiego** [comment]
Sorry, that file was wrong, the one with the image is this one: 
[generatedDocument.zip](https://github.com/open-xml-templating/docxtemplater-image-module/files/423735/generatedDocument.zip)

**Comment by edi9999** [comment]
Please also include your template and data

**Comment by edi9999** [comment]
I just added a commit to master that should be fixing the issue. 

https://github.com/open-xml-templating/docxtemplater-image-module/commit/84b0e39f75d0cb2d2c63a741d0c779d7709314b0

Can you try it out ?

**Comment by jmdiego** [comment]
Awesome!, it works perfectly, thanks a lot!!!

**Comment by edi9999** [comment]
Great, I will be releasing a new version on npm soon

**Comment by edi9999** [comment]
released in npm version 1.0.4

**Comment by jmdiego** [comment]
Great!, thanks a lot.
